src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.cs

```diff
@@ -666,6 +667,7 @@ protected override void OnPointerReleased(PointerRoutedEventArgs args)
 		protected override void OnKeyDown(KeyRoutedEventArgs args)
 		{
 			base.OnKeyDown(args);
+			((IHandleableRoutedEventArgs)args).DoNotPreventDefaultIfHandled = true;
```

Comment:
I can't say for sure from memory, but I think this will break the `Tab` and `Shift + Tab` behavior. I think the preventDefault was needed specifically to prevent the browser from handling the same key combination again when cycling focus. Can you verify that when you go to `Focus_FocusCycle` sample page, click the "Set focus on B1" button and then use `Tab` and `Shift+Tab`, the focus does not skip over two buttons at once?

Reply:
@MartinZikmund Seems to be working:

*The video shows clicking the button → press few tabs → press few SHIFT+TABs → clicking the button → press few SHIFT+TABs*

https://user-images.githubusercontent.com/31348972/133755979-3279261f-3d04-4524-a690-647be43e9435.mp4

Reply:
Interesting, I was quite sure that `preventDefault` was needed there 👀 . Can you please check also `FocusManager_FocusDirection`, click on the first `TextBox` and try `Tab` and `Shift+Tab`? If that works too, then I'm ok 😀

Reply:
@MartinZikmund Will check. It would be nice if we could write a UI test for these to prevent regressions anyway :)